app/controllers/general.php

```diff
@@ -316,16 +316,18 @@
     }
     
     $route = $utopia->match($request);
-    
-    if ($project->getId()
-        && $mode !== APP_MODE_ADMIN //TODO: add check to make sure user is admin
-        && !empty($route->getLabel('sdk.namespace', null))) { // Don't calculate console usage on admin mode
-        
-        $usage
-            ->setParam('networkRequestSize', $request->getSize() + $usage->getParam('storage'))
-            ->setParam('networkResponseSize', $response->getSize())
-            ->trigger()
-        ;
+    $appUsageStatsEnabled = App::getEnv('_APP_USAGE_STATS', 'enabled') == 'enabled';
```

Comment:
Would be nicer to add this specific portion to the condition already in place. You can have it as the first part of the `if` statement. No need to use a var for holding the condition if anyway we use that var only once in this specific code block. Same for the init callback.

Reply:
@eldadfux Not sure what you mean by init callback?

Reply:
Updated to use the existing condition and removed the variable declaration as well